veronica-starchenko/home.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+
+<html lang = "en-US">
+    <head>
+        <link rel="stylesheet" href="styles.css">
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    </head>
+
+    <body>
+        <div class="tabs">
+            <ul class="tabs-list">
+                <li><a href="home.html">Home</a></li>
+                <li><a href="index.html">About Me</a></li>
+            </ul>
+        </div>
+
+        <div>
+			<!--<h3 class="projects-heading">My Projects</h3> -->
```

Comment:
Remove comment code in all PRs you make (I know you may have copied and pasted from me, no worries!) :)